### ISSUE TITLE
[BUG] CKEditor breaks in Repeater fields when Moving Up / Down

### ISSUE THREAD

**Issue by mikenosko** [bolt/bolt#5162]
I have a repeater field setup in contentype.yml:

```
repeater:
            type: repeater
            group: repeater
            limit: 3
            prefix: "<p>This allows you to create multiple sets of fields. Use the add button at the bottom to create a new empty set.</p>"
            fields:
                repeattitle:
                    type: text
                repeatimage:
                    type: image
                    extensions: [ gif, jpg, png ]
                repeatcontent:
                    type: html
```

Now when I fill in the fields and and try to reorder them using Move Up / Move Down buttons. The repeaters reorders ok but the moved CKEditor looses the content and can't be interacted with. Trying to interact with the editor ie. pressing a button on the editor after moving produces a:

`Uncaught TypeError: Cannot read property 'getSelection' of undefined`

The relevant hidden textarea element has the content so save etc works. Reloading the page initialises the editors ok with the right content.

Details
-------

 - Relevant Bolt Version: [ 3.0 | master ]
 - Install type: [ Composer install ]
 - PHP version: [ 7.0 ]
 - Used webserver: [ Apache ] version [ 2.4.16 ]
 - For UX/UI issues: [ Chrome 49.0.2623.110 | Firefox 45.0.1 ]

**Comment by bobdenotter** [comment]
Apparently @jkazimir knows what the issue is, here! :tada:

**Comment by jkazimir** [comment]
Yep! :smile:

Basically, it's the CKE events not surviving the DOM move.  PR incoming in a bit.

**Comment by bobdenotter** [comment]
Fixed by #5171.
